src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -520,6 +522,51 @@ public String handle(
 			return gson.toJson(result);
 		}
 	}
+	
+	@Path("/api/v1/setSystem")
+	public class SetSystemEndpoint implements Route {
+		@POST
+		@Produces({ MediaType.APPLICATION_JSON })
+		@Operation(
+			summary = "System commands details", 
+			description = "Perform some system wide commands.", 
+			operationId = "systemCommand", 
+			tags = {"SDK" }, 
+			requestBody = @RequestBody(
+				description = "Command details", 
+				content = {
+					@Content(
+						mediaType = "application/json", 
+						schema = @Schema(implementation = JSONObject.class))
+				}, 
+				required = true
+			), 
+			responses = {
+				@ApiResponse(
+					responseCode = "200", 
+					description = "Successful command execution", 
+					content = @Content(
+						// TODO: Provide a specific class as value of Schema.implementation
+						schema = @Schema(implementation = Object.class)
+					)
+				),
+				@ApiResponse(responseCode = "400", description = "Bad Request")
+			}
+		)
+		@Override
+		public String handle(
+				@Parameter(hidden = true) Request request,
+				@Parameter(hidden = true) Response response
+		) {
+			try {
+				JSONObject obj = gson.fromJson(request.body(), JSONObject.class);
+			} catch (Exception e) {
+				response.status(400);
+				return e.getMessage();
+			}
+			return "";
```

Comment:
I expected a bit more in this function body, specifically:
- Parse the "command" field from the request JSON object and match it against allowed values (this was defined in the upstream yaml file): `setloglevel, reload, getloglevels, getloglevelnames, getsubsystemnames`
- Return empty array on success (instead of empty string)

```java
switch (command) {
case "setloglevel":
case "<... fill in the rest>":
  return "[]";
default:
  response.status(400);
  return "Invalid command";
}
```